### ISSUE TITLE
Can it be used to improve speaker-diarization performance for English?

### ISSUE THREAD

**Issue by anuragrawal2024** [huggingface/diarizers#3]
Hi, 

I am working on a project where I want to create speaker-aware transcripts from audios/videos, preferably using open-source solutions. I have tried so many approaches but nothing seems to work good enough out of the box.

I have tried:

1) whisperX: https://github.com/m-bain/whisperX (uses pyannote)
2) whisper-diarization: https://github.com/MahmoudAshraf97/whisper-diarization (uses Nemo)
3) AWS Transcribe
4) AssemblyAI API
5) Picovoice API

I read this in the fine-tuning google colab notebook: "The segmentation model has been trained on a combination of datasets containing mostly English and Mandarin languages. As a consequence, the performance of the speaker diarization pipeline may decrease when confronted with out-of-distribution data, such as Spanish or Japanese."

Does that mean fine-tuning won't help much in case of english speakers? My doubt is if pyannote pretrained models are already trained in English and Mandarin, how will fine-tuning using some more english data help with diarization accuracy!

I am yet to find a speaker-diarization solution that is accurate enough for english audios and am looking for suggestions for improvements.

**Comment by utility-aagrawal** [comment]
Thanks @kamilakesbi ! That's what I have been using so far and results have been below satisfactory. This is one of my test videos https://www.youtube.com/watch?v=b3ZdSWhU5vA, if you want to give it a try.

Do you have any other suggestions? From your experience, are there any audio preprocessing, hyperparameter tuning etc. that I can do to improve diarization accuracy. Thanks for your help!

**Comment by kamilakesbi** [comment]
Hi @utility-aagrawal, 

Have you identified what is causing the various approaches you have tested to fail? 

From the example you gave me, it seems that the background noise is quite high, the pipeline may not be robust to that level of noise. 

You could start by identifying what kind of errors the pipeline is making: Do you think it has a high `false_alarm_rate`, `confusion_rate` or `missed_detection_rate`?
